backend/src/main/java/com/github/rohkaz/controller/GameController.java

```diff
@@ -0,0 +1,33 @@
+package com.github.rohkaz.controller;
+
+import com.github.rohkaz.model.Game;
+import com.github.rohkaz.service.GameService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api")
+@RequiredArgsConstructor
+public class GameController {
+
+    private final GameService gameService;
+
+    @GetMapping("/games")
+    public List<Game> getAllGames() {
+        return gameService.getAllGames();
+    }
+
+    @GetMapping("/games/{id}")
+    public Game getGameByID(@PathVariable String id) {
+        return gameService.getGameByID(id);
+    }
```

Comment:
getAll and getByID are two different vertical slices.
getAll gets all Objects <- one feature
getById get the object with the given ID <- another feature